cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java

```diff
@@ -498,23 +498,22 @@ private void refreshClientOnce() {
   }
 
   private void refreshClientOnceSync(Node receiver) {
-    RaftService.Client client;
+    RaftService.Client client = null;
     try {
       client =
           getClientProvider()
               .getSyncDataClientForRefresh(receiver, RaftServer.getWriteOperationTimeoutMS());
-    } catch (TException e) {
-      return;
-    }
-    try {
       RefreshReuqest req = new RefreshReuqest();
       client.refreshConnection(req);
+    } catch (IOException ignored) {
     } catch (TException e) {
-      logger.warn("encounter refreshing client timeout, throw broken connection", e);
+      logger.info("encounter refreshing client timeout, throw broken connection", e);
```

Comment:
why use info replace warn?   generate,  exception deal bettwen warn and error.

Reply:
fixed